app/assets/javascripts/discourse/app/pre-initializers/inject-discourse-objects.js

```diff
@@ -28,7 +28,10 @@ export default {
 
     const messageBus = window.MessageBus;
     app.register("message-bus:main", messageBus, { instantiate: false });
-    ALL_TARGETS.forEach(t => app.inject(t, "messageBus", "message-bus:main"));
+
+    ALL_TARGETS.concat("service").forEach(t =>
```

Comment:
Is there a reason we can't add `service` to `ALL_TARGETS`?

Reply:
We could but I figured I would just avoid injecting everything everywhere when we don't have to.